detection_rules/rule.py

```diff
@@ -588,9 +597,33 @@ def _post_dict_transform(self, obj: dict) -> dict:
         return obj
 
     def add_related_integrations(self, obj: dict) -> None:
-        """Add restricted field related_integrations to the obj."""
-        # field_name = "related_integrations"
-        ...
+        """Add restricted field add_related_integrations to the obj"""
+
+        field_name = "related_integrations"
+        package_integrations = obj.get(field_name, [])
+
+        if self.metadata.integration:
+            packages_manifest = load_gzip_dump(get_etc_path('integration-manifests.json.gz'))
```

Comment:
can copy usage here to avoid new util funcs 

https://github.com/elastic/detection-rules/blob/c76a39796972ecde44cb1da6df47f1b6562c9770/detection_rules/ecs.py#L93